app/views/shared/_atwho_user_container.erb

```diff
@@ -21,8 +21,16 @@
     data-full-name="<%= user.full_name %>"
     data-email="<%= user.email %>"
     data-popover-html="<%= popover_html %>"
-    data-user-avatar-popover-absolute-url="<%= user_avatar_absolute_url(user, :thumb, true) %>"
+    data-user-avatar-popover-absolute-url="<%= user_avatar_absolute_url(user, :thumb, user.avatar.attached? ? true : false) %>"
```

Comment:
put result of user.avatar.attached? into a variable, you call this check 3 times in this file.
Also rename file to .html.erb

Reply:
`user.avatar.attached? ? true : false` is the same as just `user.avatar.attached?`